syntax = "proto3";

service digitalCashService {
    rpc sendToBankFromCustomer(Message) returns (Message);
	rpc ping(pingMessage) returns (ack);
}

message OldData {
	string username = 1;
	string filename =2;
	bytes data =3;
	int32 seqNo = 4;
	string replicaNode = 5;
}

message Message {
	string messageData = 1;
	int32 numberOfMoneyOrders = 2;
}

message ack {
	bool success =1;
	string message = 2;
}

message pingMessage {
	string message = 1;
}